tests/dash/test_dash_pl.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from utils.sai_utils import assert_sai_attribute_exists
+
+from dash_api.appliance_pb2 import *
+from dash_api.vnet_pb2 import *
+from dash_api.eni_pb2 import *
+from dash_api.route_pb2 import *
+from dash_api.route_rule_pb2 import *
+from dash_api.vnet_mapping_pb2 import *
+from dash_api.route_type_pb2 import *
+from dash_api.types_pb2 import *
+
+from dash_db import dash_db, DashDB
+from dash_configs import *
+from sai_attrs import *
+from swsscommon.swsscommon import (
+    APP_DASH_APPLIANCE_TABLE_NAME,
+    APP_DASH_ENI_TABLE_NAME,
+    APP_DASH_VNET_TABLE_NAME,
+    APP_DASH_VNET_MAPPING_TABLE_NAME,
+    APP_DASH_ROUTE_TABLE_NAME,
+    APP_DASH_ENI_ROUTE_TABLE_NAME,
+    APP_DASH_ROUTING_TYPE_TABLE_NAME,
+    APP_DASH_ROUTE_GROUP_TABLE_NAME,
+)
+
+DVS_ENV = ["HWSKU=DPU-2P"]
+NUM_PORTS = 2
+
+@pytest.fixture(autouse=True)
+def common_setup_teardown(dash_db: DashDB):
+    dash_db.set_app_db_entry(APP_DASH_ROUTING_TYPE_TABLE_NAME, PRIVATELINK, ROUTING_TYPE_PL_CONFIG)
+    dash_db.set_app_db_entry(APP_DASH_APPLIANCE_TABLE_NAME, APPLIANCE_ID, APPLIANCE_CONFIG)
+    dash_db.set_app_db_entry(APP_DASH_VNET_TABLE_NAME, VNET1, VNET_CONFIG)
+    dash_db.set_app_db_entry(APP_DASH_ENI_TABLE_NAME, ENI_ID, ENI_CONFIG)
+    dash_db.set_app_db_entry(APP_DASH_VNET_MAPPING_TABLE_NAME, VNET1, VNET_MAP_IP1, VNET_MAPPING_CONFIG_PRIVATELINK)
+    dash_db.set_app_db_entry(APP_DASH_ROUTE_GROUP_TABLE_NAME, ROUTE_GROUP1, ROUTE_GROUP_CONFIG)
+    # Don't set DASH_ROUTE_TABLE and DASH_ENI_ROUTE_TABLE entries here for flexibility, test cases will set them as needed
+
+    yield
+
+    dash_db.remove_app_db_entry(APP_DASH_ENI_ROUTE_TABLE_NAME, ENI_ID)
+    dash_db.remove_app_db_entry(APP_DASH_ROUTE_TABLE_NAME, ROUTE_GROUP1, OUTBOUND_ROUTE_PREFIX)
```

Comment:
@theasianpianist can we skip the route entry delete here since route group is already unbound. Same applies to the dash_vnet test cleanup.

Reply:
explicit deletion is still needed due to current limitations in the VS SAI.